src/components/Searchbar/index.jsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSearch } from '@fortawesome/free-solid-svg-icons';
+import './Searchbar.css';
+// import API from '../../services/API';
+import { getCity } from '../../redux/searchbar/searchbarSlice';
+
+const Searchbar = () => {
+  const dispatch = useDispatch();
+  // const city = useSelector((state) => state.city);
+
+  // const getCity = async (city) => {
+  //   const result = await API.getLongitudeAndLatitude(city);
+  //   return result[0];
+  // };
```

Comment:
- [ ] Kindly make sure to remove all the old and commented code from your repo it makes the code messy and hard to navigate